projects/arlas-components/src/lib/components/histogram/histogram-key.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { HistogramUtils } from 'arlas-d3';
+
+@Pipe({
+  name: 'histogramKey'
+})
+export class HistogramKeyPipe implements PipeTransform {
+
+  public transform(key: any, ...args: any[]): string {
```

Comment:
It is possible to just put histogramParams instead of args to add clearer typing